client/mixin.py

```diff
@@ -53,7 +57,7 @@ def screenshot(self) -> Screenshot:
         return self._wd.screenshot
 
     @property
-    def context(self) -> Context:
+    def windows(self) -> Windows:
         return self._wd.context
```

Comment:
Renamed `context` -> `windows`.

Reply:
Lack of ContextsApi tests and attentiveness.. 
fixed in 366fff82a1e6a5d423e57a770b430cf2306c4240